unitylibs/core/workflow/workflow-acrobat/target-config.json

```diff
@@ -8,7 +8,7 @@
     "target": ".verb-wrapper .verb-container",
     "limits": {
       "maxNumFiles": 1,
-      "maxFileSize": 1000000000,
+      "maxFileSize": 100000000,
```

Comment:
I think it should be 104857600

Reply:
Thanks @narcis-radu .This has been updated.